explainaboard/processors/named_entity_recognition.py

```diff
@@ -351,12 +328,33 @@ def _complete_features(
         :return: The features that are active (e.g. skipping training set features when
             no training set available)
         """
+        sys_features = unwrap(sys_info.features)
+        active_features = list(
+            sys_features.get_bucket_features(
+                include_training_dependent=external_stats is not None
+            )
+        )
+
+        sent_feats: list[str] = []
+        tok_feats: list[str] = []
+        for x in active_features:
+            (sent_feats if (x in sys_features) else tok_feats).append(x)
+
         for _id, dict_sysout in tqdm(enumerate(sys_output), desc="featurizing"):
             # Get values of bucketing features
             tokens = dict_sysout["tokens"]
 
             # sentence_length
             dict_sysout["sentence_length"] = len(tokens)
+            self._get_max_min_value(sys_info, "sentence_length", len(tokens))
```

Comment:
Couldn't we just do this calculation of the min and max when within the bucketing functions in `bucketing.py`? That seems much simpler than having to do it in the for loop each time we do feature extraction for each task.